tests/test_lab_evaluate.py

```diff
@@ -354,10 +354,7 @@ def test_no_model_mt_bench(cli_runner: CliRunner):
             "mt_bench",
         ],
     )
-    assert (
-        result.output
-        == "Benchmark mt_bench requires the following args to be set: ['model', 'judge-model']\n"
-    )
+    assert "Error: Missing option '--model'." in result.output
```

Comment:
Since the benchmark isn't relevant to the test anymore, worth renaming to just `test_no_model`?